stories/index.js

```diff
@@ -19,3 +19,4 @@ import "./modal";
 import "./tooltip";
 import "./multiselect";
 import "./chips";
+import "./progressbar";
```

Comment:
You should not commit this in this PR ;)

Reply:
Yeah, I opened a new branch, but this one for some reason saved here....